jetty-servlets/src/test/java/org/eclipse/jetty/servlets/IncludedGzipTest.java

```diff
@@ -90,7 +90,7 @@ public void setUp() throws Exception
 
         GzipHandler gzipHandler = new GzipHandler();
         gzipHandler.setMinGzipSize(16);
-        tester.getContext().insertHandler(gzipHandler);
+        tester.getServer().insertHandler(gzipHandler);
```

Comment:
This is not the equivalent test.  it should be `tester.getContext().insertHandler(gzipHandler);`

Reply:
perhaps test gzip in and outside of the context?